### PR TITLE
PAYARA-1016 Update Hazelcast to 3.7

### PR DESCRIPTION
**Pull request by michaelranaldo**

### CODE REVIEW

**Comment by michaelranaldo** [comment]
jenkins test please

**Comment by michaelranaldo** [comment]
dan build

**Comment by michaelranaldo** [comment]
jenkins cure all ills

**Comment by michaelranaldo** [comment]
jenkins cure all ills

**Comment by smillidge** [comment]
jenkins test please

**Comment by michaelranaldo** [comment]
jenkins test this pull request

**Comment by michaelranaldo** [comment]
jenkins test this pull request

**Comment by michaelranaldo** [comment]
jenkins test this pull request

**Comment by michaelranaldo** [comment]
jenkiins test please

File: nucleus/pom.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+W<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smillidge) [inline nucleus/pom.xml]: The 'W' shouldn't be there.

**Comment by smillidge** [comment]
there are conflicts that need to be resolved so the PR will need to be rebased against the current master

**Comment by michaelranaldo** [comment]
Jenkins test please

**Comment by smillidge** [comment]
jenkins test please

**Comment by michaelranaldo** [comment]
jenkins test please

**Comment by smillidge** [comment]
jenkins test please

**Pull request merged**